components/self_hosted_purchase_modal/self_hosted_card.tsx

```diff
@@ -78,42 +73,14 @@ export default function SelfHostedCard(props: Props) {
             <Card
                 {...dummyCardProps}
                 intl={intl}
-                topColor='#4A69AC'
                 plan={props.desiredPlanName}
-                price={`${props.desiredProduct?.price_per_seat?.toString()}`}
                 seeHowBillingWorks={seeHowBillingWorks}
-                rate='/user/month'
-                planBriefing={<></>}
-                preButtonContent={(
-                    <SeatsCalculator
```

Comment:
We're either going to need to keep using SeatsCalculator, or finish hooking up the seats change events (to set the error and the user seats) of the card so that works with self_hosted_purchase_modal. Right now, the calculator no longer updates the parent component. If we do go the route of hooking this up to use the same seats calculator, we should remove the SeatsCalculator component as well.